### DISCUSSION TITLE
”Enable villager chat AI”

### DISCUSSION THREAD

**Discussion by ThracianOne** [Luke100000/minecraft-comes-alive#611] [General]
It does not appear in the config on GitHub, however, if you go into config on later versions, you can find the boolean "enableVillagerChatAI", it is default set to false. I have enabled it, and it didn't appear to change anything, what is it supposed to do?

**Comment by Luke100000** [comment]
https://github.com/Luke100000/minecraft-comes-alive/wiki/GPT3-based-conversations

**Reply by MrXpra** [reply]
What version is it compatible with?. im using 1.16.5 version of the mod and "enableVillagerChatAI" doesnt appear in the config.

**Reply by Luke100000** [reply]
> Make sure to have 7.4.0

As stated in the doc

**Comment by Luke100000** [comment]
@sirben99 sync the wiki please ^^
